src/Microsoft.AspNet.StressFramework/StressTestRunner.cs

```diff
@@ -71,15 +82,17 @@ public class StressTestRunner : TestRunner<StressTestCase>
                     await InvokeTestMethodAsync(instance);
                 }
 
-                var stopwatch = new Stopwatch();
+                var stopwatch = Stopwatch.StartNew();
                 for (int i = 0; i < TestCase.Iterations; i++)
                 {
-                    stopwatch.Start();
+                    var context = new StressTestIterationContext(_collectors);
+                    context.BeginIteration();
 
                     await InvokeTestMethodAsync(instance);
 
-                    stopwatch.Stop();
+                    context.EndIteration();
                 }
+                stopwatch.Stop();
```

Comment:
The start/stop stuff here is arranged the way it to avoid capturing the time spend in the collectors.